src/main/kotlin/com/cognifide/gradle/aem/common/mvn/MvnModule.kt

```diff
@@ -186,6 +186,7 @@ class MvnModule(val build: MvnBuild, val descriptor: ModuleDescriptor, val proje
             workingDir.set(build.rootDir)
             args.addAll("-N", "-f", descriptor.pom.absolutePath)
             args.addAll(profileArgs)
+            if (aem.prop.flag("mvnBuild.skipTests")) args.add("-DskipTests")
```

Comment:
in GAP there is usually a DSL property that could enforce some behavior and such property could be additionally controlled by `aem.prop.x`

Reply:
you supplemented only the second config option